docs/commands/mftf.md

```diff
@@ -506,19 +506,26 @@ vendor/bin/mftf static-checks testDependencies actionGroupArguments
     
 ### `upgrade:tests`
 
-Applies all the MFTF major version upgrade scripts to test components in the given path (`test.xml`, `data.xml`, etc).
+When path argument is specified, this command will apply all the MFTF major version upgrade scripts to test components in the given path (`test.xml`, `data.xml`, etc);
```

Comment:
When the path argument is specified, this command applies all the major version MFTF upgrade scripts to the test components in the given path (`test.xml`, `data.xml`, etc).
Otherwise, it will apply all the major version MFTF upgrade scripts to all installed test components.